### PR TITLE
feat: support getting values of CSS properties

### PR DESCRIPTION
**Pull request by jimevanssfdc**

### CODE REVIEW

File: utam-compiler/src/test/resources/filter/basicFilterGetCssPropertyValue.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                  "type" : "string"
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
```
Comment (lizaiv77) [inline utam-compiler/src/test/resources/filter/basicFilterGetCssPropertyValue.json:50]: add new line

File: utam-core/src/main/java/utam/core/element/BasicElement.java [hunk 2]
```diff
@@ -60,6 +60,13 @@ public interface BasicElement extends UtamBase {
    */
   String getValue();
 
+  /**
+   * Returns the string value of the CSS property with the given name
+   * @param propertyName the name of the CSS property whose value to return
+   * @return the value of the CSS property
+   */
+  String getCssPropertyValue(String propertyName);
```
Comment (lizaiv77) [inline utam-core/src/main/java/utam/core/element/BasicElement.java:68]: pls apply format - there will be new line before param

File: utam-core/src/main/java/utam/core/element/Element.java [hunk 3]
```diff
@@ -90,6 +90,13 @@ public interface Element {
    */
   String getAttribute(String attrName);
 
+  /**
+   * get the value of the CSS property with the given name
+   * @param propertyName the name of the CSS property whose value to return
+   * @return the value of the CSS property
+   */
```
Comment (lizaiv77) [inline utam-core/src/main/java/utam/core/element/Element.java:97]: same: pls apply format - there will be new line before param

**Review by lizaiv77 — APPROVED** [review]
minor formatting, otherwise LGTM

File: utam-compiler/src/test/resources/filter/basicFilterGetCssPropertyValue.json (hunk 1 above)
Comment (jimevanssfdc) [reply utam-compiler/src/test/resources/filter/basicFilterGetCssPropertyValue.json]: Done

File: utam-core/src/main/java/utam/core/element/BasicElement.java (hunk 2 above)
Comment (jimevanssfdc) [reply utam-core/src/main/java/utam/core/element/BasicElement.java:69]: Done

**Review by lizaiv77 — APPROVED** [review]

**Pull request merged**